site/dev/common.sh

```diff
@@ -85,6 +85,26 @@ assert_not_empty () {
   fi
 }
 
+# Creates a 'nightly' version of the documentation that points to the current versioned docs
+# located at the root-level `/docs` directory.
+create_nightly () {
+  echo " --> create nightly"
+
+  # Remove any existing 'nightly' directory and recreate it
+  rm -rf docs/docs/nightly/
```

Comment:
Why is nightly not a symlink?

Reply:
Both `nightly` and `latest` both symlink the relevant `docs` directory, and copy the mkdocs.yml to update `1.5.0` to `latest` and similar with nightly.

```
/site/docs/docs/
            |
            |_nightly 
            |   |
            |   |_docs --symlink-->  /docs/docs/
            |   |_mkdocs.yaml (cp from /docs/docs/mkdocs.yaml & updated to "nightly")
            |
            |_latest
            |   |
            |   |_docs --symlink-->  ../1.5.0/docs/
            |   |_mkdocs.yaml (cp from ../1.5.0/mkdocs.yaml & updated to "latest")
            |
            |_1.5.0
            |
            |_<older_versions>